ArduCopter/adsb.cpp

```diff
@@ -34,29 +32,105 @@
 void Copter::adsb_update(void)
 {
     adsb.update();
-    adsb_handle_vehicle_threats();
+    gcs_send_text(MAV_SEVERITY_CRITICAL, "ADS-B Update");
+
+    adsb.set_enabled(1);
+    adsb.set_behavior(AP_ADSB::ADSB_BEHAVIOR_LAND);

```

Comment:
Does this hard-code the behaviour to always be Land?  Doesn't that override what the pilot has set the behaviour to?